dialog/en-us/confirm.support.dialog

```diff
@@ -0,0 +1 @@
+Would you like to create a troubleshooting package? This will upload logs to an external service and send an email to you with more information
```

Comment:
Let's be a bit more specific. I'm thinking:

> This will upload logs to the termbin dot com service and email a link to you with more information.

Reply:
@penrods Fixed